tests/baselines/reference/numericLiteralTypes1.types

```diff
@@ -44,12 +44,12 @@ type B1 = -1 | 0 | 1;
 >1 : 1
 
 type B2 = 1 | 0 | -1;
->B2 : B1
+>B2 : B2

```

Comment:
Again, this doesn't seem very useful in a test result as it is essentially a circular definition.